addons/delivery/tests/test_delivery_cost.py

```diff
@@ -451,11 +453,12 @@ def test_base_on_rule_currency_is_converted(self):
             'currency_id': currency_bells.id,
         })
 
-        self.env['res.currency.rate'].with_company(nook_inc).create({
-            'currency_id': currency_bells.id,
-            'company_rate': 0.5,
-            'inverse_company_rate': 2,
-        })
+        with freeze_time('2000-01-01'):  # make the rate is in the past
```

Comment:
```suggestion
        with freeze_time('2000-01-01'):  # Create a rate in the past.
```

Reply:
ah I meant "make sure" thank you. Though I guess yours is still clearer so using it